src/axom/slic/interface/slic.hpp

```diff
@@ -32,15 +32,29 @@ struct debug
 
 /*!
  * \brief Initializes the SLIC logging environment.
+ * \param [in] is_root Enables selective logging macros based on root.
  */
-void initialize();
+void initialize(bool is_root = true);
 
 /*!
  * \brief Checks if the SLIC logging environment is initialized.
  * \return status true if initialized, else, false.
  */
 bool isInitialized();
 
+/*!
+ * \brief Checks if we are on the root rank. Used for selective logging on root nodes.
+ * \return status true if on root, else, false.
+ */
```

Comment:
Suggestion: Consider adding a link to `initialize()` and to `setIsRoot()` (e.g. with the doxygen `\sa` or equivalent)